Src/PerformanceCollector/Perf.Shared.NetStandard20/XPlatProcessCPUPerformanceCounterNormalized.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.XPlatform
+{
+    using System;
+    using System.Diagnostics;
+    using System.Globalization;
+
+    /// <summary>
+    /// Represents normalized value of CPU Utilization by Process counter value (divided by the processors count).
+    /// </summary>
+    internal class XPlatProcessCPUPerformanceCounterNormalized : XPlatProcessCPUPerformanceCounter
+    {        
+        private readonly bool isInitialized = false;
+        private readonly int processorsCount = -1;
+
+        /// <summary>
+        ///  Initializes a new instance of the <see cref="XPlatProcessCPUPerformanceCounterNormalized" /> class.
+        /// </summary>
+        internal XPlatProcessCPUPerformanceCounterNormalized() : base()
+        {            
+            this.processorsCount = Environment.ProcessorCount;
```

Comment:
There is a code snippet below where Environment.ProcessorCount is in Try/Catch - do we need the same here?

Reply:
good catch. moved to use the utility method.